### PR TITLE
Add prompt and csvonly option

### PR DESCRIPTION
**Pull request by ffittschen**
This PR will add a prompt which asks, if the user wants to proceed and a csvonly option. When activated, only tweets contained in this csv will be deleted.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -3,3 +3,4 @@ source "https://rubygems.org"
 gem "twitter"
 gem "trollop"
 gem "dotenv"
+gem "highline"
```
Comment (mikemcquaid) [inline Gemfile]: What is Highline for?

File: twitter_delete.rb [hunk 2]
```diff
@@ -19,12 +20,14 @@
   opt :force, "Actually delete/unfavourite/unretweet tweets"
   opt :user, "The Twitter username to purge", type: :string, default: ENV["TWITTER_USER"]
   opt :csv, "Twitter archive tweets.csv file", type: :string
+  opt :csvonly, "Will only delete tweets contained in the given csv file", type: :boolean
```
Comment (mikemcquaid) [inline twitter_delete.rb]: What generates a selective CSV file?

File: twitter_delete.rb [hunk 3]
```diff
@@ -86,18 +89,24 @@ def api_call method, *args
...
-puts "Checking timeline..."
-total_tweets = [user.statuses_count, MAX_API_TWEETS].min
-oldest_tweets_page = (total_tweets / MAX_TWEETS_PER_PAGE).to_i
+puts "==> Checking timeline..."
 
-oldest_tweets_page.downto(0) do |page|
-  tweets = api_call :user_timeline, count: MAX_TWEETS_PER_PAGE, page: page
-  tweets_to_delete += tweets.reject(&method(:too_new_or_popular?))
+unless @options[:csvonly]
+    total_tweets = [user.statuses_count, MAX_API_TWEETS].min
```
Comment (mikemcquaid) [inline twitter_delete.rb]: Two-character indentation, please.

**Comment by mikemcquaid** [comment]
Hi @ffittschen! Thanks for the PR. A few thoughts and questions for you.

File: Gemfile (hunk 1 above)
Comment (ffittschen) [inline Gemfile]: I used [Highline#agree](https://github.com/JEG2/highline/blob/master/lib/highline.rb#L191) for the [prompt](https://github.com/mikemcquaid/TwitterDelete/pull/5/files#diff-5fe53d53ea29c15e6ef1bf2a468c4ce9R93)

File: twitter_delete.rb (hunk 2 above)
Comment (ffittschen) [inline twitter_delete.rb]: In my case I deleted the tweets that I want to keep from the `tweets.csv` by hand

Comment (mikemcquaid) [inline twitter_delete.rb]: Cool! I don't think I want to support that use-case for TwitterDelete though.

**Comment by mikemcquaid** [comment]
Thanks for the PR but I'd like to pass on these features at this time. I hope the tool was still useful for you, though!

**Pull request closed without merging**